src/ssr-template-helper.js

```diff
@@ -113,34 +107,40 @@ export class SsrTemplateHelper {
    */
   buildPayload_(
     request, mustacheTemplate, opt_templates, opt_attributes = {}) {
-    const ampComponent = dict({
-      'type': this.sourceComponent_,
-      'successTemplate': {
+    const ampComponent = dict({'type': this.sourceComponent_});
+
+    const successTemplate = (opt_templates && opt_templates['successTemplate'])
```

Comment:
Nit: Move 'successTemplate' to a local var to avoid typo risk.

Reply:
Done.